streamer/src/nonblocking/quic.rs

```diff
@@ -80,10 +80,11 @@ pub async fn run_server(
 ) {
     debug!("spawn quic server");
     let mut last_datapoint = Instant::now();
-    let connection_table: Arc<Mutex<ConnectionTable>> =
-        Arc::new(Mutex::new(ConnectionTable::default()));
+    let connection_table: Arc<Mutex<ConnectionTable>> = Arc::new(Mutex::new(ConnectionTable::new(
```

Comment:
This issue would have been more obvious if this variable and the corresponding parameter in `setup_connection` were both named "unstaked"
```suggestion
    let unstaked_connection_table: Arc<Mutex<ConnectionTable>> = Arc::new(Mutex::new(ConnectionTable::new(
```

Reply:
I can't add a comment at the specific spot, but the `connection_table` param in `setup_connection` still needs to be renamed

Reply:
Oh I see. The refactor tool just did it for the function local variables. I'll push another update to the PR.